homeassistant/components/sensor/zwave.py

```diff
@@ -56,8 +60,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         # Check workaround mappings for specific devices.
         if specific_sensor_key in DEVICE_MAPPINGS:
             if DEVICE_MAPPINGS[specific_sensor_key] == WORKAROUND_IGNORE:
+                logger.info('Ignoring device: %s, Index: %s',
+                            zwave._value_name(value), value.index)
                 return
 
+    if value.label == 'Previous Reading':
+        # Power measurement devices may export a 'Previous Reading' value
+        # This can be problematic, as can be seen here:
+        # https://github.com/home-assistant/home-assistant/issues/3509
+        logger.info('Ignoring device: %s', zwave._value_name(value))
```

Comment:
As far as I see it, "Previous Reading" is the only value that triggers this bug, because it gets picked up by Home Assistant. And I'm still not sure if it makes any sense to have this value in HA.

@devdelay I Fail to see any connection why the reporting should fail after ignoring it - the reporting should still update the actual Power / Energy values.

We have two options to move forward:
1) Just ignore the values, which have the same label at the same instance - As far as I can tell, this is just "Previous Reading", which isn't needed in HA. This PR fixes that.
2) Append to index to the object_id (bonus point for being backwards-compatible: only if there are multiple labels with the same name at an instance)

My vote goes to 1) ...